test/parallel/test-stream-err-multiple-callback-construction.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common');
+const stream = require('stream');
+const assert = require('assert');
+
+class TestWritable extends stream.Writable {
+  _write(_chunk, _encoding, callback) {
+    callback();
+  }
+
+  _final(callback) {
+    process.nextTick(callback);
+    process.nextTick(callback);
+  }
+}
+
+const writable = new TestWritable();
+
+writable.on('finish', common.mustCall(1));
```

Comment:
```suggestion
writable.on('finish', common.mustCall());
```

It's the same, just a nit, feel free to ignore.

Reply:
Not at laptop so I can squash later if the commits matter, unless the person merging can squash with the UI when merge